src/Sulu/Bundle/SecurityBundle/Entity/BaseUser.php

```diff
@@ -41,6 +43,7 @@
 
     /**
      * @var string
+     * @Expose
```

Comment:
Thank you for your Pull Request can you explain why this needed? The password should not be exposed to the Api as this would be a security issue.